### ISSUE TITLE
🍎 A Subtle Kind of Murder: A Murder Simulator ↔ v18.4.0

### ISSUE THREAD

**Issue by ezabba** [itchio/itch-compatibility-watchlist#26]
:rotating_light: game [A Subtle Kind of Murder: A Murder Simulator](https://almostdailymike.itch.io/a-subtle-kind-of-murder) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/7f3f16c65f685eba1a725c2f5974faa1).

:running: Any additional details can go here!

**Comment by fasterthanlime** [comment]
Aw, that's an .air game, I'm afraid the app doesn't support those at the moment :(

**Comment by ezabba** [comment]
You are right! I haven't read until the end of the page and didn't noticed
the air extension.
But maybe in the itch app should be specified that can't be played through
the app itself in a clearer way.
Nevermind, I will try with Air
Thanks

Il giorno mar 26 lug 2016 alle ore 23:25 Amos Wenger <
notifications@github.com> ha scritto:

**Comment by fasterthanlime** [comment]
> But maybe in the itch app should be specified that can't be played through
> the app itself in a clearer way.

You're right. I haven't entirely given up the idea of supporting .air files either, so we'll see which happens first!
